packages/odyssey-storybook/src/components/odyssey-mui/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,186 @@
+  },
+  {
+    name: "anchor",
+    use: "UI indicator - element contains in page anchor link",
+  },
+  {
+    name: "arrow-down",
+    use: "",
+  },
+  {
+    name: "arrow-left",
+    use: "",
+  },
+  {
+    name: "arrow-right",
+    use: "",
+  },
+  {
+    name: "arrow-up",
+    use: "",
+  },
+  {
+    name: "calendar",
+    use: "",
+  },
+  {
+    name: "check",
+    use: "UI indicator - custom checkbox",
+  },
+  {
+    name: "check",
+    use: "To show a completed process",
+  },
+  {
+    name: "chevron-down",
+    use: "UI indicator - element triggers open",
+  },
+  {
+    name: "chevron-up",
+    use: "UI indicator - element triggers close",
+  },
+  {
+    name: "close",
+    use: "To close a modal or other UI",
+  },
+  {
+    name: "close-circle-filled",
+    use: "",
+  },
+  { name: "copy", use: "To copy text" },
+  { name: "delete", use: "To delete something" },
+  { name: "download", use: "To download" },
+  {
+    name: "drag-handle",
+    use: "Element is draggable",
+  },
+  { name: "edit", use: "To edit something" },
+  {
+    name: "external-link",
+    use: "UI indicator - external link",
+  },
+  { name: "eye", use: "To show something" },
+  { name: "eye-off", use: "To hide something" },
+  { name: "filter", use: "To filter results" },
+  { name: "globe", use: "" },
+  { name: "home", use: "" },
+  {
+    name: "information-circle",
+    use: "To get information",
+  },
+  {
+    name: "information-circle-filled",
+    use: "To get information",
+  },
+  {
+    name: "notification",
+    use: "To notify the user of something",
+  },
+  { name: "overflow-vertical", use: "" },
+  {
+    name: "question-circle",
+    use: "To provide clarification",
+  },
+  {
+    name: "question-circle-filled",
+    use: "To provide clarification",
+  },
+  { name: "search", use: "To search for something" },
+  {
+    name: "settings",
+    use: "To edit user or app settings",
+  },
+  {
+    name: "subtract",
+    use: "To subtract or remove",
+  },
+  { name: "user", use: "To support a user name" },
+];
+
+export const Library = (): ReactElement => {
+  return (
+    <Table caption="Library" screenReaderCaption="Available Odyssey Icons">
+      <Table.Header>
+        <Table.Row>
+          <Table.HeaderCell>Icon</Table.HeaderCell>
+          <Table.HeaderCell>Name</Table.HeaderCell>
+          <Table.HeaderCell>Class Name</Table.HeaderCell>
+          <Table.HeaderCell>Use</Table.HeaderCell>
+        </Table.Row>
+      </Table.Header>
+      <Table.Body>
+        {meta.map(({ name, use }) => {
+          const CurrentIcon = iconDictionary[name];
+          return (
+            <Table.Row key={`${name}_row`}>
+              <Table.DataCell>
+                <CurrentIcon />
+              </Table.DataCell>
+              <Table.DataCell>{name}</Table.DataCell>
+              <Table.DataCell>{CurrentIcon.displayName}</Table.DataCell>
+              <Table.DataCell>{use}</Table.DataCell>
+            </Table.Row>
+          );
```

Comment:
Doesn't `displayName` not exist at the same values in a production build?

For development builds, it's there, but in production, I'm pretty sure those are gone completely. Babel is typically adding it to components which is why it exists at all.